libs/SalesforceSDK/src/com/salesforce/androidsdk/security/PasscodeManager.java

```diff
@@ -174,8 +178,9 @@ public int getPasscodeLengthForOrg(UserAccount account) {
      * @param account UserAccount instance.
      * @param timeout Timeout value, in ms.
      * @param passLen Minimum passcode length.
+     * @param bioAllowed If biometric Unlock is Allowed by connected App
      */
-    public void storeMobilePolicyForOrg(UserAccount account, int timeout, int passLen) {
+    public void storeMobilePolicyForOrg(UserAccount account, int timeout, int passLen, boolean bioAllowed) {
```

Comment:
Don't change this public API. Instead, add a new overloaded public API with the same name and the additional parameter and call that new API from this passing in `true` as the default value. Switch out all internal calls to the old method with the new one. Mark this deprecated too for removal in `8.0`.

```
     * @deprecated Will be removed in Mobile SDK 8.0. Use {@link blah} instead.
```

Reply:
Semantic versioning prevents us from removing this directly even though it's unlikely anyone's using this.

Reply:
👍